### ISSUE TITLE
Add BibTex entry to citation instructions

### ISSUE THREAD

**Issue by claudenirmf** [nemo-ufes/gufo#2]
The instructions for citing gUFO could include the BibTex entry as well, for example:

```
@misc{almeida2019gufo,
  title={{gUFO}: A lightweight implementation of the {Unified Foundational Ontology} ({UFO})},
  author={Almeida, Jo{\~a}o Paulo A. and Guizzardi, Giancarlo and Falbo, Ricardo Almeida and Sales, Tiago Prince},
  year={2019},
  howpublished={\url{http://purl.org/nemo/gufo}}
}
```

As far as I know, adding URLs nicely requires additional packages, so my example uses `howpublished`. The example also includes the publication year of gUFO, which is missing in the current citation instructions.

Just one additional observation: the deployed website and the latest commit do not match.

**Comment by jpalmeida** [comment]
Thanks, I added the year to the citation. However, the documentation is generated from comments in the ontology, so, I'd rather not add a BibTeX fragment to the ontology. It would require careful escaping and the rendering in different tools (including the generated html) might be problematic.

I checked the commit and the deployed website and did not find any differences. (In fact, http://purl.org/nemo/doc/gufo redirects to http://nemo-ufes.github.io/gufo/ which is served directly from the master.)
